tsdb/head.go

```diff
@@ -1615,10 +1615,19 @@ func (h *Head) MaxOOOTime() int64 {
 	return h.maxOOOTime.Load()
 }
 
+// isUninitialized returns true if the head does not yet have a MinTime or MaxTime set, false otherwise.
+func (h *Head) isUninitialized() bool {
```

Comment:
nit: it's confusing to use negatives in method names, I'd suggest to rename this method and invert its logic to `isInitialized`.